pandas/io/json/json.py

```diff
@@ -320,8 +332,21 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=True,
                 {"index": "row 2", "col 1": "c", "col 2": "d"}]}'
     """
 
+    if chunksize is not None:
+        chunksize = _validate_integer("chunksize", chunksize, 1)
+        if not lines:
```

Comment:
check ``lines`` first

Reply:
Done (I think I understood you correctly)

Reply:
this is ok i think, though should go inside jsonReader